tests/command_line/test_mcsqs_caller.py

```diff
@@ -103,5 +103,6 @@ def test_mcsqs_caller_runtime_error(self):
         struct.replace_species({"Ti": {"Ti": 0.5, "Zr": 0.5}, "Zr": {"Ti": 0.5, "Zr": 0.5}})
         struct.replace_species({"Pb": {"Ti": 0.2, "Pb": 0.8}})
         struct.replace_species({"O": {"F": 0.8, "O": 0.2}})
-        with pytest.raises(RuntimeError, match="mcsqs exited before timeout reached"):
```

Comment:
This test is failing as a different `RuntimeError` is raised:
```
>       with pytest.raises(RuntimeError, match="mcsqs exited before timeout reached"):
E       AssertionError: Regex pattern did not match.
E        Regex: 'mcsqs exited before timeout reached'
E        Input: 'mcsqs did not generate output files, is search_time sufficient or are number of instances too high?'
```

But looks like both exceptions are very similar (all pointing to timeout) as I would just change the error message here: https://github.com/materialsproject/pymatgen/blob/31f1e1fb8cc1bb517d59ea8e484965bb641c42a0/src/pymatgen/command_line/mcsqs_caller.py#L136-L160